src/site/content/en/fast/reduce-network-payloads-using-text-compression/index.md

```diff
@@ -156,3 +158,35 @@ const app = express();
 
 app.use(express.static('public'));
 </pre>
+
+## Effects on Core Web Vitals
+
+Depending the type of resource, both compression and minification can have a variety of effects on Core Web Vitals. Since both minification and compression reduce the payload size of the delivered resource, it's important to understand what Core Web Vitals are affected by these performance optimizations.
+
+### CSS
+
+When CSS resources are compressed and minified, a number of performance improvements may be observed in your Core Web Vitals metrics.
+
+- Where [First Contentful Paint (FCP)](/fcp/) is concerned, making your CSS as small as it can possibly be means that the browser will take less time to download the resource. This means that parsing of CSS can occur sooner, which should move up the rendering step.
+- Where [Largest Contentful Paint (LCP)](/lcp/) is concerned, it largely depends. If your LCP resource is specified in a CSS `background-image` property, making your CSS smaller means that the potential LCP resource can be discovered more quickly. However, LCP candidates that come from a CSS `background-image` property are not always optimal for LCP, as external CSS resources [don't benefit from the browser preload scanner](/preload-scanner/#css-background-images). If you can't move an LCP image out of CSS and into your HTML, then you should be preloading that image resource.
+
+### JavaScript
+
+JavaScript is a bit tricky, as it depends on what your JavaScript resources are providing for your website.
+
+- If your JavaScript is providing interactivity for your website, minification and compression will ensure that your website's [First Input Delay (FID)](/fid/) and [Interaction to Next Paint (INP)](/inp/) metrics may be improved during startup. However, it's also important that you're only serving the JavaScript that a given page needs, otherwise you may risk delays in responding to user inputs during that crucial startup period.
+- If your JavaScript is providing markup for a page, then the situation changes. As always, smaller payloads will download faster. In the case of JavaScript, reducing resource load time means you're going to arrive at the parsing, compiling, and execution steps sooner. This may result in faster LCP times if the client-rendered markup contains the page's LCP element. Beyond that, you should try to avoid rendering markup on the client, as [it defeats the preload scanner](/preload-scanner/#rendering-markup-with-client-side-javascript). If you can't avoid client-rendering of markup, then you should preload that render-critical piece of JavaScript so that it can arrive to the browser as fast as it possibly can.
```

Comment:
```suggestion
- If your JavaScript is providing markup for a page, then it is especially critical to allow this to happen as fast as possible. As always, smaller payloads will download faster. In the case of JavaScript, reducing resource load time means you're going to arrive at the parsing, compiling, and execution steps sooner. This may result in faster LCP times if the client-rendered markup contains the page's LCP element. Beyond that, you should try to avoid rendering markup on the client, as [it defeats the preload scanner](/preload-scanner/#rendering-markup-with-client-side-javascript).
```

I removed the last sentence - why do you think you need to preload the JS? It'll be loaded anyway. Or do you mean preload the resources in that JS so they can be discovered in the markup?